azerothcore/azerothcore-wotlk
# fix(DB/Event): Shadowfang Keep: link objects and npcs to event

## Changes Proposed:
- Link some objects and creatures from Shadowfang Keep to their event.
- https://wowwiki-archive.fandom.com/wiki/Love_is_in_the_Air

## Issues Addressed:
https://github.com/azerothcore/azerothcore-wotlk/issues/12254

## SOURCE:
Check screenshots from wowhead
https://de.classic.wowhead.com/shadowfang-keep#screenshots:id=81154

## Tests Performed:
- tested in game
- made sure that I linked the correct gameobjects by running this query:
```sql
SELECT *
FROM `game_event_gameobject` AS e
LEFT JOIN (SELECT `guid`,`id`,`map` FROM `gameobject`) AS g
ON e.guid = g.guid
LEFT JOIN (SELECT `entry`,`name` FROM `gameobject_template`) AS t
ON t.entry = g.id
WHERE e.eventEntry = 8;
```

## How to Test the Changes:
Enter Shadowfang Keep and check spawns on courtyard:
1. with event 8 active
2. with event 8 not active

## Known Issues and TODO List:

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
